VMAccess/vmaccess.py

```diff
@@ -37,10 +38,58 @@
 OutputSplitter = ';'
 SshdConfigPath = '/etc/ssh/sshd_config'
 
+# overwrite the default logger
+logger.global_shared_context_logger = logger.Logger('/var/log/waagent.log', '/dev/stdout')
```

Comment:
Do you want to log to waagent.log?

Reply:
HandlerUtil later changes the logging file to extension specific. So I have named the object global_shared_context_logger.